cmdstanpy/stanfit.py

```diff
@@ -610,17 +613,17 @@ def stan_variable(self, name: str) -> np.ndarray:
             ]
             var_dims = [dim0]
             var_dims.extend(dims)
-            return self.sample[
-                :, :, idxs[0] : idxs[len(idxs) - 1] + 1
-            ].reshape(tuple(var_dims), order='A')
+            return self.sample[:, :, idxs[0] : idxs[len(idxs) - 1] + 1].reshape(
```

Comment:
```suggestion
            return self.sample[:, :, idxs[0] : idxs[-1] + 1].reshape(
```

Last element of a sequence idx is -1